shell/apple/generate_xcode_project.command

```diff
@@ -22,11 +22,12 @@ if [[ -z "${VULKAN_SDK}" ]]; then
     if [ $v == "y" ]; then
         curl https://sdk.lunarg.com/sdk/download/latest/mac/vulkan-sdk.dmg -o vulkan-sdk.dmg
         hdiutil attach ./vulkan-sdk.dmg
-        /Volumes/vulkansdk-macos-*/InstallVulkan.app/Contents/MacOS/InstallVulkan --root $HOME/VulkanSDK --accept-licenses --default-answer --confirm-command install
-        hdiutil detach /Volumes/vulkansdk-macos-*
+        /Volumes/VulkanSDK/InstallVulkan.app/Contents/MacOS/InstallVulkan --root $HOME/VulkanSDK --accept-licenses --default-answer --confirm-command install
+        hdiutil detach /Volumes/VulkanSDK
```

Comment:
@scribam you will need to change the volume name also when you bump the Vulkan SDK version

Reply:
Ah missed that one, thanks for the info